demos/layout-grid.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
```

Comment:
General comment: Changing margin / gutter widths don't work in IE or Edge since they don't support custom properties. We should probably either hide or disable them in those browsers and add a note saying why they're disabled, similar to what we do within the ripple demo.

Furthermore, the demo page was broken in Edge on Windows 10 when I tested it in my VM:

![layout-grid-edge](https://cloud.githubusercontent.com/assets/1185269/22445845/803c0ab6-e717-11e6-9349-d4316b88bf6d.png)